src/Core/Checkout/Cart/Order/OrderConversionContext.php

```diff
@@ -2,9 +2,13 @@
 
 namespace Shopware\Core\Checkout\Cart\Order;
 
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\Framework\Struct\Struct;
 
+/**
+ * @codeCoverageIgnore
+ */
```

Comment:
Is this necessary? Structs are ignored globally

Reply:
Files ending in `Struct.php` are ignored, but therefore not a `OrderConversionContext`.